src/log.h

```diff
@@ -12,7 +12,12 @@
 #define LOG_W g_warning
 #define LOG_M g_message
 #define LOG_I g_info
-#define LOG_D g_debug
+
+#ifdef DEBUG_BUILD
```

Comment:
I'm not sure about disabling this on production builds, if anything, the ability to set dunst to debug mode and instantly have a log of everything that's going on inside the program can be very useful to debug issues.

Reply:
This is a single commit. I'll drop this.